src/wagtail_textract/models.py

```diff
@@ -12,7 +12,7 @@
 
 class Document(WagtailDocument):
     """Extra fields and methods for Document model."""
-    transcription = models.TextField(null=True)
+    transcription = models.TextField(null=True, blank=True)
```

Comment:
`null` and `blank`? Why not use `default =''` and be done? 

From django docs: 

> Avoid using null on string-based fields such as CharField and TextField.